lsp/nls/src/requests/completion.rs

```diff
@@ -166,7 +166,8 @@ fn get_identifier_path(text: &str) -> Option<Vec<String>> {
     let path = result.iter().rev().next().cloned()?;
     // Remove quotes from a record name
     fn remove_quotes(name: &str) -> String {
-        if let (Some('"'), Some('"')) = (name.chars().next(), name.chars().last()) {
+        let mut chars = name.chars();
+        if let (Some('"'), Some('"')) = (chars.next(), chars.last()) {
```

Comment:
Can we add a test as evidence of this fix? :pray: I.e. a test that fails before the fix, and succeeds after the fix

Reply:
Done, thanks!